src/modules/account/views/ViewAccountDisplayAll.vue

```diff
@@ -71,7 +71,7 @@ export default {
 
     const accounts = computed(
       () => {
-        if(walletState.currentLoggedInWallet.accounts){
+        if(walletState.currentLoggedInWallet){
```

Comment:
Tested on this line of code, it will still return null because of the else condition. I made a pull request on this.